include/boost/geometry/algorithms/detail/overlay/traversal.hpp

```diff
@@ -791,6 +794,24 @@ public :
             return false;
         }
 
+        if (is_union && cinfo.open_count == 0 && cinfo.spike_count > 0)
```

Comment:
`is_union` is `static const` so I guess it can cause warnings.

Reply:
indeed, I'll update.
Thanks for your quick review.

Reply:
Similarly here: not updated. Here it was not passed, but it is used in many places like this in this source file. And they should all be updated. 
I guess, because of the combination of conditions, there are no warnings (not sure though). In other places it was combined as well, in this source.
So also this should go to another PR.